app/api/events.py

```diff
@@ -168,7 +168,7 @@ def create(self, data, url):
         save_to_db(new_event, "Event saved")
         # set organizer
         role = Role.query.filter_by(name=ORGANIZER).first()
-        uer = UsersEventsRoles(g.user, new_event, role)
+        uer = UsersEventsRoles(user_id=creator_id, event=new_event, role=role)
```

Comment:
I did this :
In UsersEventsRoles [here](https://github.com/fossasia/open-event-orga-server/blob/development/app/models/users_events_roles.py#L17
) I am passing `g.user.id` to `user_id`, which is the same as passing `creator_id` as param.
So we don't have to add creator_id anywhere now.